src/components/MyKiva/BorrowerCarousel.vue

```diff
@@ -257,13 +258,15 @@ const onInteractCarousel = interaction => {
 	tabs.value.tabContext.selectedIndex = interaction.value;
 };
 
-onMounted(() => {
+watch(() => loans.value, () => {
 	if (!hasActiveLoans.value) {
```

Comment:
Do we need to also check `loans.value.length`? Or is the first change of this computed when the data loads?

Reply:
Is the first change when data loads...